db/controllers.js

```diff
@@ -37,8 +38,18 @@ function getArticlesById(req, res) {
         const article = response.rows[0];
         res.status(200).send(({article}));
     }).catch((error) => {
-        console.error("error in getArticlesById", error);
+        console.error(error);
+        res.status(400).send({status: 400, error: "Invalid query"});
```

Comment:
Please remove console logs from finalised code 🗑️ 
Though sending the error here currently works, we'll start having problems when more than that one error can happen and end up in the catch block (i.e. the 404 that needs to be added)
This is why a custom error handler is useful and should be added
Have a look at the notes on error handling if you're not sure how to do this https://notes.northcoders.com/courses/js-back-end/error-handling#3-testing-and-handling-errors